src/lib/components/ratio/index.svelte

```diff
@@ -136,14 +202,23 @@
   }
 
   .result-ratio {
-    align-items: start;
+    align-items: center;
     display: inline-flex;
     grid-area: number;
+    justify-content: flex-start;
```

Comment:
I wonder if we want to remove this, and let it stretch to fill the space?

Reply:
removed in https://github.com/oddbird/oddcontrast/pull/222/commits/1044fb1934534a89fbd8ef3b56960ada12e791de